tests/FSharp.Compiler.ComponentTests/Conformance/BasicGrammarElements/MemberDefinitions/OverloadingMembers/OverloadResolutionUsingFunction.fs

```diff
@@ -0,0 +1,28 @@
+open System
+let ae = new AggregateException()
+
+ae.Handle(fun e ->
+    match e with
+    | :? OperationCanceledException -> true
+    | _ -> false        
+    )
+
+ae.Handle(function
+    | :? OperationCanceledException -> true
+    | _ -> false        
+    )
+
+ae.Handle(
+    Func<exn,bool>(
+        function
+        | :? OperationCanceledException -> true
+        | _ -> false        
+    ))
+
+let m () =
```

Comment:
I think I am missing the point of this definition here, what is it here for?

Reply:
I think there should be overloaded methods, at least one of which takes a function. I may have some time today to propose some additional tests.

Reply:
Feel free to propose any runnable tests aka that I can copy and paste to this file 🙂

Reply:
@abonie Removed the useless tests. If you have some tests in mind please add these to the conversation. Thanks for the review